mup_scripts/mutransfer_lr/sp/run_openwebtext.sh

```diff
@@ -0,0 +1,54 @@
+# Single-GPU Launching
+LAUNCHER=python
+
+# Multi-GPU Launching (single node)
+#GPU=2
+#LAUNCHER=torchrun --standalone --nproc_per_node=$GPU
+
+for width in 256 512 1024 2048
+do
+    # for lr in 0.125 0.0625 0.03125 0.015625 0.0078125 0.00390625 0.001953125 0.0009765625 0.00048828125 0.000244140625 0.0001220703125 0.00006103515625
+    for lr in 0.00390625 0.001953125 0.0009765625 0.00048828125 0.000244140625 0.0001220703125 0.00006103515625 0.00003051757812
+    do
+        for seed in 1 2 3
+        do
+            head_size=64
+            n_heads=$((width / head_size))
+            out_dir="mutransfer_lr/sp/out/width${width}_depth2_seed${seed}_lr${lr}"
```

Comment:
instead please move to a dir called `mutransfer_lr_owt` to go alongside `mutransfer_lr_shakespeare_char`